### PR TITLE
C347872 Populating preview of matched records

### PR DESCRIPTION
**Pull request by MaksimTerekhov**
https://foliotest.testrail.io/index.php?/cases/view/347872

### CODE REVIEW

File: cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import Users from "../../../support/fragments/users/users";
+import BulkEditActions from "../../../support/fragments/bulk-edit/bulk-edit-actions";
+
+let user;
+const userBarcodesFileName = `userBarcodes_${getRandomPostfix()}.csv`;
+
+
+describe('bulk-edit', () => {
+  describe('csv approach', () => {
```
Comment (IhorBohdan) [inline cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js:16]: Does we need here 2 describes?

File: cypress/support/fragments/bulk-edit/bulk-edit-actions.js [hunk 2]
```diff
@@ -128,5 +128,28 @@ export default {
...
+      DropdownMenu().find(Checkbox({ name: 'barcode', checked: true })).exists(),
+      DropdownMenu().find(Checkbox({ name: 'patronGroup', checked: true })).exists(),
+      DropdownMenu().find(Checkbox({ name: 'username', checked: true })).exists(),
+      DropdownMenu().find(Checkbox({ name: 'email', checked: false })).exists(),
+      DropdownMenu().find(Checkbox({ name: 'expirationDate', checked: false })).exists(),
+    ]);
+  },
+
+  verifyCheckedDropdownMenuItem() {
+    cy.do(DropdownMenu().find(Checkbox({ name: 'firstName'})).click());
```
Comment (IhorBohdan) [inline cypress/support/fragments/bulk-edit/bulk-edit-actions.js:147]: Move checkbox to const

File: cypress/support/fragments/bulk-edit/bulk-edit-actions.js [hunk 3]
```diff
@@ -128,5 +128,28 @@ export default {
...
+    cy.expect(MultiColumnListHeader('First name').absent());
+  },
+
+  verifyUncheckedDropdownMenuItem() {
+    cy.do(DropdownMenu().find(Checkbox({ name: 'email'})).click());
```
Comment (IhorBohdan) [inline cypress/support/fragments/bulk-edit/bulk-edit-actions.js:152]: Move checkbox to const

File: cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import permissions from '../../../support/dictionary/permissions';
+import TopMenu from '../../../support/fragments/topMenu';
+import FileManager from '../../../support/utils/fileManager';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import testTypes from '../../../support/dictionary/testTypes';
+import devTeams from '../../../support/dictionary/devTeams';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import Users from "../../../support/fragments/users/users";
+import BulkEditActions from "../../../support/fragments/bulk-edit/bulk-edit-actions";
+
```
[the remark this answers is not in the document]
Comment (MaksimTerekhov) [reply cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js:10]: done

File: cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js (hunk 1 above)
Comment (MaksimTerekhov) [reply cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js:16]: Yes, it is need temporarly for allure

File: cypress/support/fragments/bulk-edit/bulk-edit-actions.js (hunk 3 above)
Comment (MaksimTerekhov) [reply cypress/support/fragments/bulk-edit/bulk-edit-actions.js:152]: I used it only 1 time
I am not sure about move this code to upper scope

File: cypress/support/fragments/bulk-edit/bulk-edit-actions.js (hunk 2 above)
Comment (penfriend) [reply cypress/support/fragments/bulk-edit/bulk-edit-actions.js:147]: here is makes sense to move  DropdownMenu() to const

Comment (MaksimTerekhov) [reply cypress/support/fragments/bulk-edit/bulk-edit-actions.js:147]: done

File: cypress/support/fragments/bulk-edit/bulk-edit-actions.js (hunk 3 above)
Comment (MaksimTerekhov) [reply cypress/support/fragments/bulk-edit/bulk-edit-actions.js:152]: done

File: cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js (hunk 1 above)
Comment (penfriend) [reply cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js:16]: That's needed for the convenience in the reporter's display

**Review by penfriend — APPROVED** [review]

File: cypress/support/fragments/bulk-edit/bulk-edit-actions.js [hunk 5]
```diff
@@ -128,5 +129,28 @@ export default {
...
+    ]);
+  },
+
+  verifyCheckedDropdownMenuItem() {
+    cy.do(dropdownMenu.find(Checkbox({ name: 'firstName'})).click());
+    cy.expect(MultiColumnListHeader('First name').absent());
+  },
+
+  verifyUncheckedDropdownMenuItem() {
+    cy.do(dropdownMenu.find(Checkbox({ name: 'email'})).click());
```
Comment (IhorBohdan) [inline cypress/support/fragments/bulk-edit/bulk-edit-actions.js:153]: Move checkbox to const

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**